core-model/src/main/java/com/sequenceiq/cloudbreak/domain/environment/RegistrationType.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.domain.environment;
+
+/**
+ * The type of the network creation.
+ * EXISTING: An existing network has been attached.
+ * CLOUDBREAK: The network created by Cloudbreak.
```

Comment:
I don't think this comment is necessary. The enum is enough self explanatory. And we can already see a disadvantage of commenting code: it references a thing that does not exist anymore.